bsdploy/bootstrap-files/pkg.conf

```diff
@@ -1 +1,8 @@
 ASSUME_ALWAYS_YES: YES
+REPO_AUTOUPDATE: NO
```

Comment:
What's the reasoning? You can't install old packages anyway. It only makes sense in context of the proxy stuff afaics.